Dockerfile

```diff
@@ -9,10 +21,11 @@ COPY requirements.txt requirements_dev.txt requirements_all_ds.txt ./
 RUN pip install -r requirements.txt -r requirements_dev.txt
 RUN if [ "x$skip_ds_deps" = "x" ] ; then pip install -r requirements_all_ds.txt ; else echo "Skipping pip install -r requirements_all_ds.txt" ; fi
 
-COPY . ./
-RUN npm install && npm run bundle && npm run build && rm -rf node_modules
+COPY . /app
+RUN rm -rf /app/client/*
```

Comment:
I think this is not necessary. The COPY from the frontend-builder will override this anyway.

Reply:
`/app` seems a little bit sensitive because `/app` is mount `.` on host. I tried `npm install` into /app in frontend-builder, but it disappeared in the resulting container. I'd like to talk about this in #3201